app/views/api/v1/accounts/articles/_article.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+json.id article.id
+json.category_id article.category_id
+json.title article.title
+json.content article.content
+json.description article.description
+json.status article.status
+json.account_id article.account_id
+json.portal_id article.portal_id
+json.views article.views
+json.author_id article.author_id
```

Comment:
Should we include author instead of author_id. Similarly portal instead of portal_id?